documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='con-optimization-proposals-{context}']
+
+= Optimization proposals overview
+
+An _optimization proposal_ is a summary of proposed changes that would produce a more balanced Kafka cluster, with partition workloads distributed more evenly among the brokers. 
+Each optimization proposal is based on the set of xref:con-optimization-goals-{context}[optimization goals] that were used to generate it.
+
+Optimization proposals always satisfy all the configured hard goals, although not necessarily all the soft goals. Hard goals are ignored if `skip_hard_goal_check: true` is specified in the `KafkaRebalance` custom resource.
+
+An optimization proposal contains the following information, in JSON, about the pending cluster rebalance:
+
+* An optimal mapping of partitions to brokers, generated by the Analyzer component of Cruise Control
+
+* Changes to the partition leaders
+
+* Statistics on the cluster rebalance (before and after)
+
+You can use the information to decide whether to implement the cluster rebalance, or change the optimization goals. 
+
+NOTE: All optimization proposals are dry runs: you cannot implement a cluster rebalance without first generating an optimization proposal. 
+There is no limit to the number of optimization proposals that can be generated.
+
+[discrete]
+== Contents of optimization proposals
+
+The following table explains the properties contained in an optimization proposal:
+
+[cols="2*",options="header",stripes="none",separator=¦]
+|===
+
+m¦JSON property
+¦Description
+
+m¦numIntraBrokerReplicaMovements
+¦Description
```

Comment:
This indicates how many partition replicas in total will be transferred between the brokers in the Kafka Cluster.

Reply:
I think this is "how many partition replicas in total will be transferred between the disks of an individual brokers, since it is "intra"  instead "inter", at least that is how the goals define the meaning, but I could be mistaken! I concede to you

If this is between disks then this isn't supported as I never added the auto configuration for JBOD!

Reply:
Oh good point! Got my definitions mixed up, thanks for checking Kyle.